### PR TITLE
View widget change

### PR DESCRIPTION
**Pull request by khushbu-maurya**

### CODE REVIEW

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

File: src/main/webapp/app/entities/flair-bi/tableview/flair-bi-display-visualization-tableview.html [hunk 1]
```diff
@@ -25,5 +27,8 @@ <h4>Execute At: <span class="label label-default">{{vm.reportData.taskExecuted |
                 </tbody>
             </table>
         </div>
+        <div id="content-{{vm.id }}" style="height: 600px;">
```
Comment (johibkhan2) [inline src/main/webapp/app/entities/flair-bi/tableview/flair-bi-display-visualization-tableview.html:30]: add css in scss file

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Pull request merged**